.gitignore

```diff
@@ -4,3 +4,4 @@ coredns.exe
 build/
 release/
 vendor/
+.idea/
```

Comment:
Please don't add IDE specific items to the git ignore.  Only build artifacts should go here.